src/i_flmusic.c

```diff
@@ -462,6 +476,7 @@ static void I_FL_BindVariables(void)
 #if defined(_WIN32)
     "soundfonts",
 #else
+    "./soundfonts:"
```

Comment:
I think we can even extend this change to the `_WIN32` case, because even here the `soundfonts` directory is in general relative to the executable directory.

Reply:
I think it already works that way in `_WIN32`. I've never had a problem with it and no user complaints.

Reply:
Right. I was thinking about this "simplification", but we could just leave the `_WIN32` code as is:

```patch
--- a/src/i_flmusic.c
+++ b/src/i_flmusic.c
@@ -473,10 +473,9 @@ static const char **I_FL_DeviceList(void)
 static void I_FL_BindVariables(void)
 {
     M_BindStr("soundfont_dir", &soundfont_dir,
-#if defined(_WIN32)
-    "soundfonts",
-#else
-    "./soundfonts:",
+    "./soundfonts",
+#if !defined(_WIN32)
+    ":"
     // RedHat/Fedora/Arch
     "/usr/share/soundfonts:"
     // Debian/Ubuntu/OpenSUSE
```

Reply:
I like old version more.